vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow-integration-tests/src/main/java/com/vaadin/flow/component/richtexteditor/tests/RichTextEditorSetHtmlValuePage.java

```diff
@@ -1,50 +0,0 @@
-package com.vaadin.flow.component.richtexteditor.tests;
-
-import com.vaadin.flow.component.html.Div;
-import com.vaadin.flow.component.html.NativeButton;
-import com.vaadin.flow.component.richtexteditor.RichTextEditor;
-import com.vaadin.flow.data.value.ValueChangeMode;
-import com.vaadin.flow.router.Route;
-
-import java.util.stream.Stream;
-
-@Route(value = "vaadin-rich-text-editor/set-html-value")
-public class RichTextEditorSetHtmlValuePage extends Div {
```

Comment:
I'm not 100% sure about this test. It was introduced by the fix for a BFP (https://github.com/vaadin/vaadin-rich-text-editor/issues/175), where the problem was that setting a value via `AsHtml.setValue` does not trigger a change event via `RichTextEditor.addValueChangeListener`.

However I'm not sure why this is testing every single value change mode, since value change modes should only affect changes coming from the client. Decided to remove this, and instead added a test for setting an initial HTML value to the new `RichTextEditorValuePage`, to cover the issue from the BFP.